docs/contents/getting_started/index.md

```diff
@@ -37,6 +37,12 @@ Before downloading PDF.js please take a moment to understand the different layer
 
 ## Download
 
+You can use PDF.js by including this file from [jsDelivr](https://www.jsdelivr.com/package/npm/pdfjs-dist):
```

Comment:
Remove the text from the download section, list it in the separate section below.